abis/UniswapV2Factory.json

```diff
@@ -0,0 +1,193 @@
+[
+    {
```

Comment:
nitpick: Just, since we discussed today that we might have to use both V2 and V3, probably is a good idea to create  all the ABIs in different folders (`uniswapV2` and `uniswapV3`)